docs/onboarding/onboarding-feature.md

```diff
@@ -0,0 +1,173 @@
+- [onboarding feature manifest](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/onboarding.fml.yaml)
+
+The onboarding feature is a **first run experiment**. 
+- [for information on first run experiment](https://experimenter.info/mobile-first-run-experiments)
+- [for information on localization](https://experimenter.info/mobile-messaging/#localization-of-messages)
+
+# Card definition
+| Attribute              | Type                        | Description                                  | Notes                                                                                                         |       
+|------------------------|-----------------------------|----------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| card-type              | Card Type                   | The type of the card                         | [See defined cards types](#card-types)                                                                                     |
+| title                  | Free text  or   Resource ID | The title text displayed to the user         |                                                                                                               | 
+| body                   | Free text  or  Resource ID  | The message text displayed to the user       | May contain linkable text                                                                                     | 
+| link-text (optional)   | Free text or  Resource ID   | The text to link from the ‘body’ text        | Must match the linkable text from the ‘body’ exactly e.g. body: This is a policy link, link-text: policy link |
+| image-res              | Resource ID                 | The resource ID of the image to be displayed |                                                                                                               | 
+| image-is-illustration  | Boolean                     | True if the image type is an illustration    |                                                                                                               |
+| ordering               | Integer                     | Used to sequence the cards                   | The system used for ordering is counting in tens, e.g. 10, 20, 30…                                            |
+| primary-button-label   | Free text  or  Resource ID  | The text to display on the primary button    |                                                                                                               | 
+| secondary-button-label | Free text or  Resource ID   | The text to display on the secondary button  |                                                                                                               | 
+
+## Card types
+- default-browser
+- sync-sign-in
+- notification-permission
+
+# Default cards
+
+## Default browser card
```

Comment:
Some explanatory text needs to go after "Default Cards", but not the tables.

I'd put these tables in an appendix or even another page.